app-text/poppler/poppler24-24.11.0.recipe

```diff
@@ -9,9 +9,9 @@ LICENSE="Apache v2
 	GNU GPL v2
 	GNU GPL v3
 	poppler_24.06.0-2_copyright"
-REVISION="2"
+REVISION="3"
```

Comment:
reset to 1

Reply:
I know :) libVersions need to be adjusted also :)